src/components/Reports/GoalCalculator/CalculatorSettings/Steps/OneTimeGoalsStep/OneTimeGoalsStepForm/OneTimeGoalsStepForm.tsx

```diff
@@ -32,15 +32,9 @@ interface OneTimeGoalsFormValues {
   }>;
 }
 
-interface OneTimeGoalsStepFormProps {
-  formikProps: FormikProps<OneTimeGoalsFormValues>;
-}
-
-export const OneTimeGoalsStepForm: React.FC<OneTimeGoalsStepFormProps> = ({
-  formikProps,
-}) => {
+export const OneTimeGoalsStepForm: React.FC = () => {
   const { t } = useTranslation();
-  const { values, setFieldValue } = formikProps;
+  const { values, setFieldValue } = useFormikContext<OneTimeGoalsFormValues>();
```

Comment:
So you can use `useFormikContext` and it will grab the Formik that is the closest to this component?

Reply:
Yep! `<Formik>` creates a React context component, and `useFormikContext` pulls the formik state from that context. It's really handy when your form is split up across multiple components.